### PR TITLE
chore: Clarify @section in graindocs

### PR DESCRIPTION
**Pull request by av8ta**
Hi grain team, the new docs on docs are great but I found the last sentence confusing. I hope this change is both correct and clarifies!

### CODE REVIEW

**Review by phated — APPROVED** [review]
Looks good to me! I'll wait for another approval and merge. Thanks for the feedback and improvements ❤️

**Review by ospencer — APPROVED** [review]
Looks great, thank you @av8ta 😄

**Comment by av8ta** [comment]
Sweet :)

**Pull request merged**
